installer.sh.in

```diff
@@ -1058,8 +1061,21 @@ failed to create filesystem $fstype on $dev!\ncheck $LOG for errors." ${MSGBOXSI
         # Mount rootfs the first one.
         [ "$mntpt" != "/" ] && continue
         mkdir -p $TARGETDIR
-        echo "Mounting $dev on $mntpt ($fstype)..." >$LOG
-        mount -t $fstype $dev $TARGETDIR >$LOG 2>&1
+        if [ "$fstype" = "btrfs" ]; then
+            # Setup subvolume on BTRFS partitions
+            echo "Mounting $dev on $mntpt ($fstype) to create subvolume @..." >$LOG
+            mount -t $fstype $dev $TARGETDIR >$LOG 2>&1
+            btrfs subvolume create $TARGETDIR/@ >$LOG 2>&1 
```

Comment:
This is cool. But, how about setting multi subvolume like `@home` and `@snapshot`? does btrfs just create `@` subvolume enough?